src/app/login/login.component.ts

```diff
@@ -36,6 +40,22 @@ export default class LoginComponent {
     pass: fb.control<string>('', Validators.required),
   });
 
+  constructor() {
+    effect(() => {
+      if (this.failedLogin()) {
+        setTimeout(() => {
+          this.passwordInput?.nativeElement?.focus();
+        }, 0);
+      }
+    });
+  }
+
+  ngOnInit() {
+    setTimeout(() => {
+      this.passwordInput?.nativeElement?.focus();
+    }, 0);
+  }
+
```

Comment:
Use ngAfterContentInit instead of using the timeout

Also lets put effects as named variables 

`focusIfFailed = effect(() => this.failedLogin() && this.passwordInput().nativeElement.focus());`